engine/src/engine.ts

```diff
@@ -226,6 +232,13 @@ export default class Engine {
     if (this.options.factionVariant === undefined) {
       this.options.factionVariant = "standard";
     }
+    if (this.options.auction && !this.isVersionOrLater("4.8.0")) {
```

Comment:
```suggestion
    if (this.options.auction === true) {
```

To cover an edge case: when the game is created on BGS with old options (auction is a boolean), but not started, so engine is not initialized yet. 

Then gaia-project is updated to new version

Then players join & game is started: it will have the new version but old options (`auction` is a boolean)